player/lua/console.lua

```diff
@@ -1998,6 +1985,10 @@ mp.register_event('log-message', function(e)
             terminal_styles[e.level])
 end)
 
+mp.register_event('shutdown', function ()
```

Comment:
What is the purpose of this?

Reply:
It unsets `open` if you `set load-console no` while the console is open.